modules/rosa-hcp-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc
+
+[id="rosa-hcp-architecture_{context}"]
+= ROSA with HCP architecture
+
+In ROSA with HCP, the ROSA service hosts a highly-available, single-tenant OpenShift control plane. The hosted control plane is deployed across 3 availability zones with at least 2 API server instances and three etcd instances. It is only accessible from the customer's VPC subnets. Customers access the hosted control plane through AWS PrivateLink. Customers can create a ROSA with HCP cluster with or without an internet-facing API server.
```

Comment:
@wanghaoran1988 @mjlshen  Can you please help on this section?

Reply:
There are guaranteed to be 2 API server instances and no more. Also, these two sentences seem to be in conflict:
> It (the hosted control plane) is only accessible from the customer's VPC subnets.

and

> Customers can create a ROSA with HCP cluster with or without an internet-facing API server.

I think something more like this might be better, though I'm not 100% sure if we call our options public/private or public/PrivateLink for HCP

```suggestion
In ROSA with HCP, the ROSA service hosts a highly-available, single-tenant OpenShift control plane. The hosted control plane is deployed across 3 availability zones with 2 API server instances and three etcd instances. Customers can create a ROSA with HCP cluster with a public or private API server. Private API servers are only accessible from the customer's VPC subnets. Customers access the hosted control plane through an AWS PrivateLink VPC Endpoint.
```

Reply:
Here is a take: 

In ROSA with HCP, the ROSA service hosts a highly available single tenant OpenShift control plane. The hosted control plane is deployed across 3 availability zones with 2 API Server instances and 3 etcd instances. Cluster's worker nodes are deployed in customer's account VPC private subnets. The API Server is exposed to customer's VPC hosting worker nodes through AWS Private Link. Customers can create a ROSA with HCP cluster with or without an internet-facing API Server. 

- I'd suggest to not call PrivateLink as substitute for Private as we have in Classic STS. This will confuse.
- Please use numerals (1,2,3) or words (one, two, three) consistently
- This is about architecture so don't mention how it's created by default etc. Write this in different section.